packages-common/pluggable-widgets-tools/package.json

```diff
@@ -102,15 +105,20 @@
     "webpack-merge": "^4.2.2"
   },
   "devDependencies": {
-    "@mendix/generator-widget": "^8.9.0",
-    "cross-env": "^5.2.0",
-    "fs-extra": "9.0.0",
+    "@mendix/generator-widget": ">=8.9.0",
+    "async": "^3.2.0",
+    "async-mutex": "^0.2.2",
+    "cross-env": "^7.0.2",
+    "fs-extra": "^9.0.0",
+    "rimraf": "^3.0.2",
```

Comment:
we can use shelljs's `rm -rf` iso rimraf

Reply:
We can't use this, because we are using `rimraf` in `package.json` `prepare` script.

Reply:
Changed it to `shx`